astronomer/providers/dbt/cloud/operators/dbt.py

```diff
@@ -0,0 +1,80 @@
+import time
+from typing import Any, Dict, Optional
+
+from airflow import AirflowException
+from airflow.providers.dbt.cloud.hooks.dbt import DbtCloudHook
+from airflow.providers.dbt.cloud.operators.dbt import DbtCloudRunJobOperator
+from airflow.utils.context import Context
+
+from astronomer.providers.dbt.cloud.triggers.dbt import DbtCloudRunJobTrigger
+
+
+class DbtCloudRunJobOperatorAsync(DbtCloudRunJobOperator):
+    """
+    Executes a dbt Cloud job asynchronously. Trigger the dbt cloud job via worker to dbt and with run id in response
+    poll for the status in trigger.
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:DbtCloudRunJobOperator`
```

Comment:
did you check in docstring if actually this ref work?

Reply:
Yes it is working referring the sync version operator
![Uploading Screenshot 2022-09-10 at 10.37.54 AM.png…]()